aws/resource_aws_backup_vault_test.go

```diff
@@ -68,6 +68,15 @@ func TestAccAwsBackupVault_withTags(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_backup_vault.test", "tags.left", "right"),
 				),
 			},
+			{
+				Config: testAccBackupVaultWithUpdateTags(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAwsBackupVaultExists("aws_backup_vault.test", &vault),
+					resource.TestCheckResourceAttr("aws_backup_vault.test", "tags.%", "2"),
+					resource.TestCheckResourceAttr("aws_backup_vault.test", "tags.foo", "bar"),
+					resource.TestCheckResourceAttr("aws_backup_vault.test", "tags.fizz", "buzz"),
+				),
+			},
```

Comment:
You may want to test adding/removing tags too

Reply:
Thanks. Added test case.

```
$ make testacc TEST=./aws TESTARGS='-run=TestAccAwsBackupVault_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAwsBackupVault_ -timeout 120m
=== RUN   TestAccAwsBackupVault_basic
=== PAUSE TestAccAwsBackupVault_basic
=== RUN   TestAccAwsBackupVault_withKmsKey
=== PAUSE TestAccAwsBackupVault_withKmsKey
=== RUN   TestAccAwsBackupVault_withTags
=== PAUSE TestAccAwsBackupVault_withTags
=== CONT  TestAccAwsBackupVault_basic
=== CONT  TestAccAwsBackupVault_withTags
=== CONT  TestAccAwsBackupVault_withKmsKey
--- PASS: TestAccAwsBackupVault_basic (33.29s)
--- PASS: TestAccAwsBackupVault_withKmsKey (66.19s)
--- PASS: TestAccAwsBackupVault_withTags (74.41s)
PASS
ok  	github.com/terraform-providers/tmp-terraform-provider-aws/aws	74.504s
```